Source/JavaScriptCore/jit/JITOperations.cpp

```diff
@@ -4601,6 +4601,17 @@ JSC_DEFINE_JIT_OPERATION(operationReallocateButterflyToGrowPropertyStorage, char
     OPERATION_RETURN(scope, reinterpret_cast<char*>(result));
 }
 
+JSC_DEFINE_JIT_OPERATION(operationThrowTypeErrorFrom, void, (JSGlobalObject* globalObject))
+{
+    VM& vm = globalObject->vm();
+    CallFrame* callFrame = DECLARE_CALL_FRAME(vm);
+    JITOperationPrologueCallFrameTracer tracer(vm, callFrame);
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    throwVMTypeError(globalObject, scope, typedArrayBufferHasBeenDetachedErrorMessage);
+    OPERATION_RETURN(scope);
+}
+
```

Comment:
Not necessary if we just jump to slow C++ code

Reply:
I think we still need that for DFG and FTL, right?